src/test/java/uk/gov/hmcts/reform/sendletter/SampleData.java

```diff
@@ -56,6 +57,15 @@ public static LetterWithPdfsRequest letterWithPdfsRequest() {
         );
     }
 
+    public static uk.gov.hmcts.reform.sendletter.entity.Letter letterEntityWithSentToPrintAt(
```

Comment:
can we not reuse `letterEntity`method and then set `letterEntity` where you are calling this ?

Reply:
we can. I'll update

Reply:
done